docs/pages/server-access/guides/host-user-creation.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Configure Teleport to create host users
+description: How to configure Teleport to automatically create transient host users.
+---
+
+Teleport's SSH service can be configured to automatically create local Unix users
+upon login.
+
+Host users created by Teleport are transient and will be deleted at the end of an SSH
+session.
+
```

Comment:
I think it would be good to add a short paragraph about why transient users are useful, especially in relation to manually adding OS logins or automatically adding them via Teleport's [PAM support](https://goteleport.com/docs/server-access/guides/ssh-pam/#create-local-unix-users-on-login).